<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeQuiz</class>
 <widget class="QDialog" name="MakeQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>问题设计</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="Ques">
      <property name="placeholderText">
       <string>请输入问题描述</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Opt1">
      <property name="placeholderText">
       <string>请输入选项一</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Opt2">
      <property name="placeholderText">
       <string>请输入选项二</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Opt3">
      <property name="placeholderText">
       <string>请输入选项三</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Opt4">
      <property name="placeholderText">
       <string>请输入选项四</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Opt5">
      <property name="placeholderText">
       <string>请输入选项五</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>121</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>正确答案</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Che1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Che2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Che3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Che4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Che5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>119</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>出题</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
